library/std/src/lib.rs

```diff
@@ -495,6 +496,9 @@ pub mod task {
 #[stable(feature = "futures_api", since = "1.36.0")]
 pub mod future;
 
+#[unstable(feature = "async_stream", issue = "79024")]
+pub mod stream;
```

Comment:
Why this is not just a reexport of `core::stream`?

Reply:
I anticipated `std`-specific extension might arise, and they did! -- this now contains the `AssertUnwindSafe` implementation for `Stream`, which can only be implemented as part of `std` because that's where it's defined.

Reply:
oh actually; you pointed out they should be implemented on the other submodules -- I think that's indeed better and that would indeed enable this to just be a re-export. Good point!

Reply:
okay, updated to just make it a re-export!